contracts/src/interfaces/IPayerReport.sol

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+/**
+ * @title IPayerReport
+ * @notice Interface for the PayerReport contract handling usage reports and batch settlements.
+ */
+interface IPayerReport {
+    //==============================================================
+    //                             STRUCTS
+    //==============================================================
+
+    struct UsageReport {
+        address originatorNode;
+        uint256 startingSequenceID;
+        uint256 endingSequenceID;
+        uint256 lastMessageTimestamp;
+        uint256 reportTimestamp;
+        bytes32 reportMerkleRoot;
+        address[] payers;
```

Comment:
With the merkle root, would we actually need the `payers` and `amountsSpent` on this struct?

Reply:
That's correct, we don't. Each node can compute the lists off-chain.